src/testHelpers.ts

```diff
@@ -0,0 +1,29 @@
+import { Plugin } from "prosemirror-state";
```

Comment:
This file was pulling Cypress definitions into a Jest environment, so I've moved it to be within this project. The `index.ts` and its helpers should probably be organised more carefully in another PR.

Reply:
I agree about the organising, happy to leave it to another PR.